components/core/src/os/users/windows.rs

```diff
@@ -67,6 +67,11 @@ mod tests {
 
     use super::*;
 
+    #[test]
+    fn get_uid_of_current_user() {
+        assert!(get_uid_by_name(&get_current_username().unwrap()).is_some());
```

Comment:
Functionally equivalent, but I find this clearer (and prefer to avoid unwraps)
```suggestion
        assert!(get_current_username().map(|s| get_uid_by_name(&s)).is_some())
```